db/migrate/20200410153801_remove_original_id_foreign_keys.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class RemoveOriginalIdForeignKeys < ActiveRecord::Migration[5.2]
+  def change
+    # Removing these because there is not a big downside and the upside is we can
+    # still use original_id across systems if we don't have to meet this requirement.
+    remove_foreign_key "forms", "forms"
```

Comment:
I don't think I understand the motivation for this, can you clarify?

Reply:
```
    # The only thing this foreign key is used for is for tracking which original object
    # copies point to so that they are not duplicated during subsequent copies.
    # (e.g. so the same question is not copied twice when two different forms that use
    # it are imported to a mission).
    # But if there is a constraint on the foreign key then it means we can't use it across 
    # different servers, which could end up being useful.
    # So let's remove the constraint.
```